CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+website/content/docs/contributors/community.md
```

Comment:
Could we link to the site at httpp://heronstreaming.io/docs/contributors/community/ or  http://twitter.github.io/heron/docs/contributors/community/ or would that not work?